arangograph/single-sign-on.md

```diff
@@ -25,17 +44,98 @@ can sync the information about user resources in real-time:
 - `GET /Users` - Lists all users.
 - `GET /Users/:user_id` - Get details for a given user ID.
 - `POST /Users` - Invite a new user to ArangoGraph.
-- `PATCH /Users/:user_id` - Update a given user ID.
+- `PUT /Users/:user_id` - Update a given user ID.
 - `DELETE /Users/:user_id` - Delete a specified user ID.
 
-Organization administrators can generate an API key for a specific organization.
-The API key consists of a key and a secret. Using this key and secret as the
-Basic Authentication Header in SCIM provisioning, you can access the APIs and
+ArangoGraph organization administrators can generate an API key for a specific organization.
+The API token consists of a key and a secret. Using this key and secret as the
+Basic Authentication Header (Basic Auth) in SCIM provisioning, you can access the APIs and
 manage the user resources.
 
-To learn how to generate a new API key, see the [API Keys](my-account.html#api-keys) section.
+To learn how to generate a new API key in the ArangoGraph Dashboard, see the
+[API Keys](my-account.html#api-keys) section.
 
 {% hint 'info' %}
 When creating an API key, it is required to select an organization from the
 list.
-{% endhint %}
\ No newline at end of file
+{% endhint %}
+
+## Configure SAML 2.0 and SCIM using Okta
+
+You can enable SSO for your ArangoGraph organization using Okta as an Identity
+Provider (IdP). For more information about Okta, please refer to the
+[Okta Documentation](https://help.okta.com/en-us/Content/index.htm?cshid=csh-index){:target="_blank"}.
+
+### Create the SAML app integration in Okta
+
+1. Go to Okta and select **Applications** from the left sidebar menu.
```

Comment:
Can we make Okta a link to their dashboard or whatever they have to make clearer where to go?

Reply:
I don't think Okta has generic dashboard links, but we can add that they need to sign into Okta first